addon/utils/ajax-helpers.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+/**
+ * Helper functions for asynchronous behavior
+ *
+ * @module ember-osf
+ * @module utils
+ */
+
+/**
+ * Helper functions to support AJAX requests
+ * @class ajax-helpers
+ */
+
+/**
+ * Performs any additional authorization config required, for the configured authorization type.
+ * Allows manual AJAX requests to be authorization-agnostic.
+ *
+ * Primarily used to set XHR flags on manual AJAX requests, for cookie based authorization.
+ * @method ajaxAuth
+ * @param {Object} options
+ * @returns {Promise}
+ */
+function ajaxAuth(options) {
```

Comment:
I'd been going for similarity to jquery method names, but this is an easy change to make. 

(if someone quibbles about authentication vs authorization, we can always rename the method to `zxcvbnm` later)